app/views/reports/setup.js.erb

```diff
@@ -21,7 +21,6 @@ $('#defined_reports_step_2').html("<%= raw escape_javascript(render(:partial =>
 $('#report_selection').hide();
 $('#defined_reports, #defined_reports_step_2').show();
 // $('#defined_reports_step_1').dialog('close');
-$("select").selectpicker()
```

Comment:
@Stuart-Johnson @kyle-glick Why do we need to make them selectpickers? I say that should go into a separate story, fix later. This is a bug on production.